src/core/Object3D.js

```diff
@@ -411,7 +419,15 @@ class Object3D extends EventDispatcher {
 
 		this.updateWorldMatrix( true, false );
 
-		_m1.copy( this.matrixWorld ).invertTransform();
+		if ( this.matrixAutoUpdate ) {
```

Comment:
Can we move this check in `Matrix.invert`? I.e. if `Matrix.matrixAutoUpdate`, then use `invertTransform`, otherwise old `invert` (but keep `invert` and `invertTransform` functions).

Reply:
`matrixAutoUpdate` is a property of `Object3D`. I would not start adding update flags to the math classes...

Reply:
I did not noticed that is not the `Matrix4` file, sorry...